iterator.go

```diff
@@ -195,7 +199,7 @@ func (RootLookuper) newMsgFromParts(qName string, qType uint16) *dns.Msg {
 	msg := new(dns.Msg)
 	msg.SetQuestion(qName, qType)
 	msg.RecursionDesired = false
-	msg = msg.SetEdns0(65000, false)
+	msg = msg.SetEdns0(2048, false)
```

Comment:
What's this change about?

Reply:
65k was crazy and I am too lazy for now to do a proper doubleup